### ISSUE TITLE
WCAG 4.1.2: Ensures aria-hidden elements are not focusable nor contain focusable elements (span[aria-hidden="true"])

### ISSUE THREAD

**Issue by Jinssi** [Jinssi/Keilamenu#1]
#### Issue
`Ensures aria-hidden elements are not focusable nor contain focusable elements` ([aria-hidden-focus](https://accessibilityinsights.io/info-examples/web/aria-hidden-focus))
#### Target application
[(1) Feed | LinkedIn](https://www.linkedin.com/feed/)
#### Element path
.update-components-actor__sub-description--untruncated > span[aria-hidden="true"]
#### Snippet
`<span aria-hidden="true"><!---->Promoted by<span class="white-space-pre"> </span><a class="THGLZiYzqWGWKdttMOgKMIHkarZmDticds " target="_self" href="https://www.linkedin.com/company/cisco/" data-test-app-aware-link=""><span><span><!---->Cisco<!----></sp...`
#### How to fix
    Fix all of the following:
      Focusable content should have tabindex="-1" or be removed from the DOM
#### Environment
Microsoft Edge version 132.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.44.0 (axe-core 4.9.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
